contracts/near/eth-client/rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-10-08
+stable-2022-01-13
```

Comment:
```suggestion
stable-2022-01-19
```

Reply:
❯ cargo build --target wasm32-unknown-unknown --release
info: syncing channel updates for 'stable-2022-01-19-x86_64-apple-darwin'
error: no release found for 'stable-2022-01-19'